confd/tests/compiled_templates/explicit_peering/keepnexthop-global/bird.cfg

```diff
@@ -68,13 +67,21 @@ template bgp bgp_template {
 
 # ------------- Global peers -------------
 
+
+
 # For peer /global/peer_v4/172.19.4.87
 protocol bgp Global_172_19_4_87 from bgp_template {
+  ttl security off;
+  multihop;
   neighbor 172.19.4.87 as 64533;
   source address 10.192.0.2;  # The local address we use for the TCP connection
   next hop keep;
 }
 
+
```

Comment:
Wonder where these newlines are coming from. Would be nice to find out and get rid of them, if possible.

Reply:
Looks like these probably came from something I was doing during debugging. I didn't realize we diffed with `--ignore-blank-lines` so was expecting the test to tell me about any formatting changes which never came. I've gone ahead and fixed up all the whitespace changes.